Src/VsVimShared/VsVimShared.projitems

```diff
@@ -40,6 +40,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\InlineRename\InlineRenameListenerFactory.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\InlineRename\InlineRenameUtil.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Implementation\IntelliCode\IntelliCodeCommandTarget.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)implementation\intellicode\IntelliCodeExtensionAdapter.cs" />

```

Comment:
nit: the capitalization of this line is different rom the others

Reply:
odd behavior, this is auto-generated code 🤷
I'll manually fix.